### PR TITLE
✅ Implementation + Test cases done

### PR DESCRIPTION
**Pull request by yashksaini-coder**
This PR adds the implementation of the previously stated requirements.

> The current scrap.py feature retrieves text from [cl1p.net](https://cl1p.net/) and displays it in the > console output. We want to expand this feature by adding two new functionalities, which have already 
> been implemented in other files of the project:
> Copy to Clipboard
>
> Write to File
>
> Task:
> Integrate the copy-to-clipboard and file write functionalities into the [scrap.py]

✅ Fixes: #141

can you please review this PR, and also assign it to me with proper labels. @this-is-yaash

### CODE REVIEW

**Comment by yashksaini-coder** [comment]
Here are the results:- 

[sneaky-package-141.webm](https://github.com/user-attachments/assets/3e58798f-a788-4eb1-b2b2-b831c0f60577)

File: .vscode/settings.json
Comment (this-is-yaash) [inline .vscode/settings.json:1]: Is this file necessary?

File: package/output.txt
Comment (this-is-yaash) [inline package/output.txt:1]: if you are using this `output.txt` file for evaluating with the test cases, then relocate it under `test/` directory. That would good! (if that's the case)

File: package/scrap.py
Comment (this-is-yaash) [inline package/scrap.py:1]: After testing in local env, 
```py
from scrap import grab
grab("test", "test.txt")
```
- Automatically copies to clipboard
- With [2]arg, it writes the file in the root dir of execution.

But the I want this to execute differently,

```py
from scrap.grab import show, clip, write
show("test") # display console
clip("test") # copy to clipboard
write("test", "text.txt") #write in the root dir of execution
```

> even though the feature is different that utilize internet, i want to stick with the traditional execution such as `show`, `clip`, `write`.

so far, your implementation works good!
just change the structure of execution that matches with the current requirement!

File: tests/test_scrap.py
Comment (this-is-yaash) [inline tests/test_scrap.py:1]: Just change the test cases as per the current requirement mentioned of `scrap.py`.

> I'll raise the levels for this one!

File: output.txt
Comment (this-is-yaash) [inline output.txt:1]: remove it

**Review by this-is-yaash — COMMENTED** [review]
please make the necessary changes and raise the PR!

File: .vscode/settings.json
Comment (yashksaini-coder) [reply .vscode/settings.json:1]: It got merged when I reconfigured the test cases, but it is of value as it tell vscode what test framework to use for testing. It can be setup locally if no

File: package/output.txt
Comment (yashksaini-coder) [reply package/output.txt:1]: The output.txt is generated in the same directory where the function is called, the path cna be changed if the user gives the input.

File: tests/test_scrap.py
Comment (yashksaini-coder) [reply tests/test_scrap.py:1]: I wrote the test cases for all the subprocess checks and the grab, and output to the text file.

File: output.txt
Comment (yashksaini-coder) [reply output.txt:1]: okh
